report/report.tex

```diff
@@ -97,29 +97,24 @@
 %----------------------------------------------------------------------------------------
 
 \section{Introduction}
+In the globalized economy of today, consumers are presented with a range of products to choose from when shopping. This can present some difficulty, as quality of the product is not always clear from its presentation. The customer review seeks to solve this by providing feedback on the purchase. Still, determining the value of a review can be difficult, as reviews can be misleading or provide little information.
 
-\lettrine[nindent=0em,lines=3]{L} orem ipsum dolor sit amet, consectetur adipiscing elit.
-\blindtext % Dummy text
-
-\blindtext % Dummy text
+Voting systems set in place to identify helpful reviews help consumers circumvent this issue, yet these reviews still get lost in the sheer number of unlabeled reviews. Our machine learning model aims to further filter reviews by analyzing verbosity, sentiment and other aspects, to discover what makes a review helpful.
 
 %------------------------------------------------
 
 \section{Methods}
+We have made use of the agile and iterative research approach proposed in Nunamaker et al., to continuously gain feedback on the activities involved in our process.
```

Comment:
Remember to reference the Nunamaker paper :DD

Reply:
To be done later 8)

Reply:
Bare at vi husker det :DDD